src/main/CMakeLists.txt

```diff
@@ -1,12 +1,17 @@
-add_executable(clusters_ultimate energy.cpp)
-target_include_directories(clusters_ultimate PRIVATE ${CMAKE_SOURCE_DIR})
-target_include_directories(clusters_ultimate PRIVATE ${PROJECT_SOURCE_DIR}/../external/kdtree)
+add_executable(energy energy.cpp)
+target_include_directories(energy PRIVATE ${CMAKE_SOURCE_DIR})
+target_include_directories(energy PRIVATE ${CMAKE_SOURCE_DIR}/../external/kdtree)
+target_include_directories(energy PRIVATE ${CMAKE_SOURCE_DIR}/io_tools/)
+target_include_directories(energy PRIVATE ${CMAKE_SOURCE_DIR}/bblock/)
+target_include_directories(energy PRIVATE ${CMAKE_SOURCE_DIR}/potential/1b/)
+target_include_directories(energy PRIVATE ${CMAKE_SOURCE_DIR}/tools/)
 
-target_link_libraries(clusters_ultimate bblock)
-target_link_libraries(clusters_ultimate io_tools)
-target_link_libraries(clusters_ultimate 1b)
+target_link_libraries(energy bblock)
+target_link_libraries(energy io_tools)
+target_link_libraries(energy tools)
+target_link_libraries(energy 1b)
 
-install(TARGETS clusters_ultimate
+install(TARGETS energy
```

Comment:
I wouldn't recommend simply calling an executable "energy", there should be a more self explanatory name.